README.md

```diff
@@ -81,6 +89,18 @@ test\_\_human_join
 The coverage improvement is +100%.
 The coverage has been improved, because `_human_join` used to not be tested, while the new `test__human_join` considers every branch to be taken.
 
+#### Frank
+
+test_from_str_failures
+
+[commit](https://github.com/tthijm/sep/commit/51a7b35a9b4af8058cc9d513085b02306ae63a43)
+
+![](./assets/parse_rgb_number_before.png)
+
+![](./assets/parse_rgb_number_after.png)
+
+The coverage of the function has improved from the originaly 78% to 100%, since the other two branches (1 and 3) were also covered.
```

Comment:
```suggestion
The coverage of the function has improved from the original 78% to 100%, since the other two branches (1 and 3) were also covered.
```